Gemfile

```diff
@@ -283,6 +283,8 @@ gem 'StreetAddress', require: "street_address"
 
 gem 'recaptcha', require: 'recaptcha/rails'
 
+gem 'loofah', ' ~> 2.2.1'
```

Comment:
Is this the right way to force this, or should I just do it in the lock file?

Reply:
Since we version-control the lock file either is fine for bumping a version. This way makes it more explicit we're requiring a version-range for a specific reason, so slightly better.